newspaper/nlp.py

```diff
@@ -116,13 +116,12 @@ def split_words(text):
         return None
 
 
-def keywords(text):
+def keywords(text, num_keywords=10):
```

Comment:
This is an improvement! but can you go even further and make this configurable via the config file?
https://github.com/codelucas/newspaper/blob/624a6ea77d55575f77bd1f6697034dc575b05c7d/newspaper/configuration.py#L23